app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +91,621 @@ class Patient extends React.Component {
     return (
       <React.Fragment>
         <Row id="monitoree-details-header">
-          <Col className="mt-1">
-            <div className="h3">
+          <Col sm={12}>
+            <h3>
               <span className="pr-2">
                 {`${this.props.details.first_name ? this.props.details.first_name : ''}${
                   this.props.details.middle_name ? ' ' + this.props.details.middle_name : ''
                 }${this.props.details.last_name ? ' ' + this.props.details.last_name : ''}`}
               </span>
               {this.props?.dependents && this.props?.dependents?.length > 0 && <BadgeHOH patientId={String(this.props.details.id)} location={'right'} />}
-            </div>
+            </h3>
           </Col>
-          <Col md="auto">
+          <Col sm={12}>
             <div className="jurisdiction-user-box">
-              <Row id="jurisdiction-path">
-                <Col>
-                  <b>Assigned Jurisdiction:</b> {this.props.jurisdiction_path || '--'}
-                </Col>
-              </Row>
-              <Row id="assigned-user">
-                <Col>
-                  <b>Assigned User:</b> {this.props.details.assigned_user || '--'}
-                </Col>
-              </Row>
+              <div id="jurisdiction-path">
+                <b>
+                  <span className="d-none d-md-inline">Assigned</span> Jurisdiction:
```

Comment:
this actually shows the `Assigned` portion at breakpoints medium and above, so it hides it on small screens where there is not enough room to have both works.  If you resize the window you will see how this works

Reply:
Yeah I see how it works, but I mean why is this necessary for Assigned Jurisdiction and not Assigned User - when the jurisdiction can get very long? Just was curious where the requirement came from rather than just wrapping the jurisdiction name.

Reply:
In Mike's mockup's the note is to save space, which makes sense because the field value and key are longer.  You can also still understand shortening to "Jurisdiction" but not "User" because that is much more vague.  @mrnosal am I missing anything else?

Reply:
Correct - it hides "Assigned" at breakpoints smaller than md (<767px). I picked this because "Assigned Jurisdiction" is a longer label than "Assigned User", and jurisdiction names tend to be longer than user id's. By hiding the word "Assigned" at the small sizes, it gives a little extra room to keep the block on the same row. The jurisdiction name will wrap automatically anyway.